9_report.yml

```diff
@@ -127,7 +127,7 @@ targets:
       fig_ind=target_name,
       config_fig_yml = '9_report/cfg/config_fig.yml',
       exceed_cfg_yml = '9_report/cfg/exceedance_threshold.yml',
-      preds_ind='3_forecast/out/preds_loadest.rds.ind',
```

Comment:
point scored for remake and our use of it - so simple to switch inputs! =)

Reply:
so easy!